interfaces/matlab/toolbox/@ThermoPhase/speciesCharges.m

```diff
@@ -0,0 +1,12 @@
+function ch = speciesCharges(tp)
+% SPECIESCHARGES  Get the elementary charge of the species.
```

Comment:
I would suggest using essentially the same description as you have for the Python property, e.g. "Get the array of species charges".